src/app/shared/components/notification-card/notification-card.component.ts

```diff
@@ -32,27 +23,14 @@ export class NotificationCardComponent implements OnInit {
   constructor(private readonly elementRef: ElementRef) {}
 
   public ngOnInit(): void {
-    this.setBackground()
+    this.setClassName()
   }
 
   public remove(): void {
     this.onRemoveCallback(this.notification)
   }
 
-  private setBackground(): void {
-    const statusColor: string = this.getStatusColor()
-    this.elementRef.nativeElement.style.background = `linear-gradient(${GRADIENT_DEGREE}, ${statusColor}, ${statusColor} ${STATUS_COLOR_WIDTH}, ${BACKGROUND_COLOR} ${STATUS_COLOR_WIDTH}, ${BACKGROUND_COLOR})`
-  }
-
-  private getStatusColor(): string {
-    switch (this.notification.statusCode) {
-      case StatusCode.BAD:
-        return ERROR_COLOR
-      case StatusCode.WARNING:
-        return WARNING_COLOR
-      case StatusCode.GOOD:
-      case StatusCode.UNKNOWN:
-        return INFO_COLOR
-    }
+  private setClassName(): void {
+    this.elementRef.nativeElement.classList.add(this.notification.statusCode.toLocaleLowerCase())
```

Comment:
This is only called once upon init. Instead of modifying the native element, a host binding be used, which will give a reactive binding. Another plus is that it will remove the need for the constructor parameter (and the explicit constructor) and `ngOnInit`.

E.g.:
```typescript
class NotificationCard {
  @HostBinding('class')
  protected get statusCodeClassName(): string {
    return this.notification.statusCode.toLocaleLowerCase()
  }
}
```

Reply:
Done